<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmWindowLayout</class>
 <widget class="QFrame" name="frmWindowLayout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QLabel" name="lblCurrentWindow1">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="lblCurrentWindow2">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QTableWidget" name="tblPlot2">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>150</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1000000</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="gridStyle">
        <enum>Qt::SolidLine</enum>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>60</number>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>40</number>
       </attribute>
       <row>
        <property name="text">
         <string>r0</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>r1</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>r2</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>r3</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>r4</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>r5</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>r6</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>r7</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>r8</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>r9</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>c0</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>c1</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>c2</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>c3</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>c4</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>c5</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>c6</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>c7</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>c8</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>c9</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lblWindow">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>Window</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0" rowspan="4">
      <widget class="QListWidget" name="lstWindow">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>200</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="autoScroll">
        <bool>true</bool>
       </property>
       <property name="tabKeyNavigation">
        <bool>false</bool>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>true</bool>
       </property>
       <property name="dragEnabled">
        <bool>false</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="movement">
        <enum>QListView::Static</enum>
       </property>
       <property name="flow">
        <enum>QListView::TopToBottom</enum>
       </property>
       <property name="isWrapping" stdset="0">
        <bool>false</bool>
       </property>
       <property name="resizeMode">
        <enum>QListView::Fixed</enum>
       </property>
       <property name="layoutMode">
        <enum>QListView::SinglePass</enum>
       </property>
       <property name="uniformItemSizes">
        <bool>false</bool>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="selectionRectVisible">
        <bool>false</bool>
       </property>
       <property name="currentRow">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="lblPlot">
       <property name="maximumSize">
        <size>
         <width>10000</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>Plot Layout</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QTableWidget" name="tblPlot1">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>150</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1000000</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="gridStyle">
        <enum>Qt::SolidLine</enum>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>60</number>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>40</number>
       </attribute>
       <row>
        <property name="text">
         <string>r0</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>r1</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>r2</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>r3</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>r4</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>r5</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>r6</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>r7</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>r8</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>r9</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>c0</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>c1</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>c2</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>c3</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>c4</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>c5</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>c6</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>c7</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>c8</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>c9</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
